### PR TITLE
Fixed remaining cops

### PR DESCRIPTION
**Pull request by ares**
spaces left of `{` fail in jenkins

### CODE REVIEW

File: app/helpers/policies_helper.rb [hunk 1]
```diff
@@ -88,7 +88,7 @@ def show_partial_wizard(step)
   end
 
   def previous_link(form)
-    previous = content_tag(:span, :class => 'glyphicon glyphicon-chevron-left') {}
+    previous = content_tag(:span, :class => 'glyphicon glyphicon-chevron-left'){}
```
[the remark this answers is not in the document]
Comment (ares) [reply app/helpers/policies_helper.rb]: erm, jenkins says 

```
12:04:42 /var/lib/workspace/workspace/test_plugin_matrix/database/sqlite3/ruby/2.4/slave/fast/plugin/app/helpers/policies_helper.rb:91:80: C: Layout/SpaceBeforeBlockBraces: Space detected to the left of {.
12:04:42     previous = content_tag(:span, :class => 'glyphicon glyphicon-chevron-left') {}
12:04:42                                                                                ^
```

so either hound or jenkins ignores our configuration?

[the remark this answers is not in the document]
Comment (xprazak2) [reply app/helpers/policies_helper.rb]: I would change it to:
```ruby
previous = content_tag(:span, :class => 'glyphicon glyphicon-chevron-left')
```
I think we do not need to pass an empty block

File: .rubocop.yml [hunk 2]
```diff
@@ -128,6 +128,9 @@ Rails/ReversibleMigration:
   Exclude:
        - 'db/migrate/20150111085317_polymorph_asset.rb'
 
+Layout/SpaceBeforeBlockBraces:
+  Enabled: false
```
Comment (xprazak2) [inline .rubocop.yml]: I would not add this, we have spaces everywhere and removing empty block in `policies_helper` should take care of the last offence

**Comment by ares** [comment]
updated based on your recommendations

**Comment by xprazak2** [comment]
Merging, thanks!

**Pull request merged**